iron-overlay-behavior.html

```diff
@@ -477,7 +478,6 @@
           this._focusNode.focus();
         }
       } else {
-        this._focusNode.blur();
```

Comment:
The overlay is not displayed anymore at this point (it is `opened = false`, which means `display: none`, hence not focusable).